library/templates/dcp_mappluto.yml

```diff
@@ -51,15 +40,15 @@ dataset:
       - dcp_colp
       - lpc_historic_districts
       - lpc_landmarks
-      - dcp_cdboundaries
+      - dcp_cdboundaries_wi
       - dcp_censustracts
```

Comment:
Should the dcp_censustracts/blocks be updated to reflect the distinction between 2010 and 2020 that we now make? dcp_ct2010, dcp_ct2010_wi, dcp_ct2020, dcp_ct2020_wi, dcp_cb2010, etc..